backend/src/main/resources/application.yml

```diff
@@ -61,4 +61,18 @@ management:
     livenessState:
       enabled: true
     readinessState:
-      enabled: true
\ No newline at end of file
+      enabled: true
+server:
+  port: 8081
```

Comment:
Wirds Probleme mit dem Helm Deployment machen? Oder reicht es dort die backend ports auf 8081 zu ändern? @eiskasten  
Anderer Port, weil sonst FE und BE nach außen gleichen port zu verwenden versuchen. 

Aber ich könnte eventuell die Ports gar nicht exposen. Dann kann man den compose nicht wirklich für die Entwicklung verwenden.

Reply:
Glaub, das ist wieder hinfällig, weil ich im compose die url sowie nicht dynamisch setzen kann, wenn ich außerhalb von docker zugreife, weil ich dann im Browser sowas "http://backend:8081/api/v1" habe und das geht nat nicht